### PR TITLE
Jw epic question update

### PR DESCRIPTION
**Pull request by jwiel86**
For more information, please refer to:  https://www.pivotaltracker.com/story/show/133538113

### CODE REVIEW

File: app/views/dashboard/protocols/form/study_form_sections/_readonly_form.html.haml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.form-group.row
+  = form.label :selected_for_epic, t(:protocols)[:studies][:information][:push_to_epic], class: 'col-lg-2 control-label required'
+  .col-lg-10
+    %span.epic_selected.display_epic_answers
+      = form.object.selected_for_epic ? t(:constants)[:yes_select] : t(:constants)[:no_select]
+      - if form.object.selected_for_epic
+        = form.hidden_field :selected_for_epic, value: true
+      - elsif !form.object.selected_for_epic
```
Comment (jayhardee9) [inline app/views/dashboard/protocols/form/study_form_sections/_readonly_form.html.haml]: Could `else` work just as well here?

File: app/assets/javascripts/protocol_form.js.coffee [hunk 2]
```diff
@@ -56,108 +55,36 @@ $(document).ready ->
...
-              add_and_check_visual_error_on_submit(research_active_dropdown)
-              add_and_check_visual_error_on_submit($(restrict_sending_dropdown))
-              return false
-  ######## End of send to epic study question logic ##############
+  determine_study_type = (answers) ->
+    array_values = new Array()
+    for k,v of answers
+      array_values.push(v)
+    nil_value = $.inArray('', array_values) > -1
+    if array_values[0] == 'true' || nil_value == false
```
Comment (jayhardee9) [inline app/assets/javascripts/protocol_form.js.coffee]: I'd change `nil_value == false` to `!nil_value`

File: app/assets/javascripts/protocol_form.js.coffee [hunk 3]
```diff
@@ -56,108 +55,36 @@ $(document).ready ->
...
+      $.ajax
+        type: 'POST'
+        data: answers
+        url: "/study_type/determine_study_type_note"
+        success: ->
+          $('#study_type_note').show()
```
Comment (wtholt) [inline app/assets/javascripts/protocol_form.js.coffee]: What happens if this errors out? Should we do some error handling here on AJAX error?

File: app/controllers/dashboard/protocols_controller.rb [hunk 4]
```diff
@@ -90,6 +89,7 @@ def show
   end
 
   def new
+    @action = params['action']
```
Comment (wtholt) [inline app/controllers/dashboard/protocols_controller.rb]: You can access the controller action using #action_name (http://stackoverflow.com/questions/15841617/if-action-name-rails)

**Pull request merged**